src/components/common/Searchbar/index.js

```diff
@@ -0,0 +1,11 @@
+import ReactDOM from "react-dom";
```

Comment:
Looks like we didn't quite remove this file like we meant to given the github nonesense.